lib/Attribute.js

```diff
@@ -56,12 +59,12 @@ function Attribute(schema, name, value) {
   else if (this.type.name === 'list'){
 
     if(value.type) {
+      if (value.list === undefined || value.list[0] === undefined){
+        throw new errors.SchemaError('No list schema given for attribute:' + this.name );
```

Comment:
```suggestion
        throw new errors.SchemaError(`No list schema given for attribute: ${this.name}`);
```

Super picky, I know. But PR #465 is doing this, so I think from now on we should do it throughout the code.